src/Symfony/Component/Finder/Iterator/RecursiveDirectoryIterator.php

```diff
@@ -31,6 +31,11 @@ class RecursiveDirectoryIterator extends \RecursiveDirectoryIterator
      */
     private $rewindable;
 
+    // these 3 properties take part of the performance optimization to avoid redoing the same work in all iterations
+    private $rootPath;
+    private $subPath = false;
```

Comment:
null is the return value of $this->getSubPath() on hhvm, thus requiring `false` here as "not set" placeholder